app/code/core/Mage/Core/Helper/Data.php

```diff
@@ -651,17 +651,28 @@ public function jsonEncode($valueToEncode, $cycleCheck = false, $options = array
      * Decodes the given $encodedValue string which is
      * encoded in the JSON format
      *
+     * switch added to prevent exceptions in json_decode
+     *
      * @param string $encodedValue
      * @return mixed
      */
     public function jsonDecode($encodedValue, $objectDecodeType = Zend_Json::TYPE_ARRAY)
     {
         switch (true) {
-            case (null === $encodedValue)  : $encodedValue = 'null'; break;
-            case (true === $encodedValue)  : $encodedValue = 'true'; break;
-            case (false === $encodedValue) : $encodedValue = 'false'; break;
-            case ('' === $encodedValue)    : $encodedValue = '""'; break;
-            default    : // do nothing
+            case (null === $encodedValue):
+                $encodedValue = 'null';
+                break;
+            case (true === $encodedValue):
+                $encodedValue = 'true';
+                break;
+            case (false === $encodedValue):
+                $encodedValue = 'false';
+                break;
+            case ('' === $encodedValue):
+                $encodedValue = '""';
+                break;
+            default:
+                // do nothing
```

Comment:
This is just whitespace change which replaces the original update from https://github.com/OpenMage/magento-lts/pull/83. I think whitespace-only changes should be avoided unless there is good reason, particularly when the author is not the same.

Reply:
if this reflexs what is in magento now, I think its better to keep the coding style they do use in case of future changes.

Reply:
it was included into the patch, i didn't add anything from myself. nevertheless new code use zend code style guide https://framework.zend.com/manual/1.11/en/coding-standard.coding-style.html#coding-standards.coding-style.control-statements.switch